db/sequence_allocator.go

```diff
@@ -99,7 +102,7 @@ func (s *sequenceAllocator) releaseSequenceMonitor() {
 			for {
 				select {
 				case <-s.reserveNotify:
-				case <-time.After(s.releaseSequenceWait):
+				case <-timer.C:
```

Comment:
Timers only fire once. I think we'll need to move the timer declaration into the first <-s.reserveNotify case, or call `Reset()` on it at that point.

Reply:
Makes sense, resetting timer would be a better choice. Changes are available in latest revision.